cmd/satellite/open_stream.go

```diff
@@ -70,9 +70,9 @@ func NewOpenStreamCommand() *cobra.Command {
 
 	command.Flags().StringVarP(&mode, "mode", "m", "udp", "The proxy mode to use. One of [udp].")
 	command.Flags().StringVar(&listenHost, "listen-host", "127.0.0.1", "The host to listen for packets on.")
-	command.Flags().Uint16Var(&listenPort, "listen-port", 6000, "The port to listen for packets on.")
+	command.Flags().Uint16Var(&listenPort, "listen-port", 6000, "Stellar CLI listens packets to be sent to a satellite on this port.")
```

Comment:
Let's still use our normal grammar. So `The port stellar listens for packets on. Packets on this port will be sent to the satellite.`

`The port stellar sends packets to. Packets from the satellite will be sent to this port.`